packages/simulation-new/src/actors/Robot.ts

```diff
@@ -106,14 +99,14 @@ export class Robot {
     const actyx = this.actyx;
     // TODO: on tab being accidentally closed, restore previously accepted task
     const query = `
-            PRAGMA features := subQuery interpolation
-
-            FROM '${ProtocolName}' ORDER DESC FILTER _.type = '${Events.WaterRequested.type}'
-            LET done_events := FROM \`${ProtocolName}:{_.requestId}\` FILTER _.type = '${Events.WateringDone.type}' END ?? []
-            FILTER !IsDefined(done_events[0])
-            LET accepted_events := FROM \`${ProtocolName}:{_.requestId}\` FILTER _.type = '${Events.HelpAccepted.type}' FILTER IsDefined(_.robotId) FILTER _.robotId = '${this.id}' END
-            FILTER IsDefined(accepted_events[0])
-        `;
+        PRAGMA features := subQuery interpolation
+
+        FROM '${ProtocolName}' ORDER DESC FILTER _.type = '${Events.WaterRequested.type}'
+        LET done_events := FROM \`${ProtocolName}:{_.requestId}\` FILTER _.type = '${Events.WateringDone.type}' END ?? []
+        FILTER !IsDefined(done_events[0])
+        LET accepted_events := FROM \`${ProtocolName}:{_.requestId}\` FILTER _.type = '${Events.HelpAccepted.type}' FILTER IsDefined(_.robotId) FILTER _.robotId = '${this.data.id}' END
```

Comment:
For the sake of readability I think we can give this query some "room to breathe" by adding a few newlines

Reply:
agreed